[Blog] Weeks 25 and 26 (#136)
packit-service/packit.dev: content/posts/June-2020.md

* We are now parsing events from Koji about the progress of the builds and report them (by [Franta](https://github.com/lachmanfrantisek/), [#657](https://github.com/packit-service/packit-service/pull/657)).
* We are now able to listen to Gitlab hooks and parse the (comment & push) events (by [Shreyas](https://github.com/shreyaspapi), [#629](https://github.com/packit-service/packit-service/pull/629), [#671](https://github.com/packit-service/packit-service/pull/671)).
* Log pages have been restructured by linking SRPM build logs instead of including them (by [Hunor](https://github.com/csomh), [#641](https://github.com/packit-service/packit-service/pull/641)).

## Week 25 (June 15th - June 19th)

### packit

> * For failures related to RebaseHelper, packit can show more information about the error (by [@csomh](https://github.com/csomh), [#865](https://github.com/packit-service/packit/pull/865)).

Reviewer: ```suggestion
* Packit logs more information for RebaseHelper errors (by [@csomh](https://github.com/csomh), [#865](https://github.com/packit-service/packit/pull/865)).
```

### packit-as-a-service

 * Obtaining builds via packit [API endpoint](https://prod.packit.dev/api/copr-builds) is now significantly quicker (by [@IceWreck](https://github.com/IceWreck), [#674](https://github.com/packit-service/packit-service/pull/674)).
 * We now have an API endpoint for obtaining results from testing farm (by [@IceWreck](https://github.com/IceWreck), [#678](https://github.com/packit-service/packit-service/pull/678)).

>  * We finished the the trigger for creating bugzilla. Therefore packit is now able to create bugzilla for accepted PR's in centos (by [@jpopelka](https://github.com/jpopelka), [#662](https://github.com/packit-service/packit-service/pull/662)).

Reviewer: ```suggestion
 * We finished the implementation of the trigger to create bugs in Bugzilla. Therefore packit is now able to create bugzilla for accepted PR's in CentOS (by [@jpopelka](https://github.com/jpopelka), [#662](https://github.com/packit-service/packit-service/pull/662)).
```

## Week 26 (June 22th - June 26th)

### packit

>  * packit now supports custom repositories and packages for building in copr and has an option to preserve project or list in on home page [@lachmanfrantisek](https://github.com/lachmanfrantisek), [#872](https://github.com/packit-service/packit/pull/872).

Reviewer: ```suggestion
 * packit now supports using custom repositories and packages when building in Copr and has options to preserve the project and to list it in on the Copr home page [@lachmanfrantisek](https://github.com/lachmanfrantisek), [#872](https://github.com/packit-service/packit/pull/872).
```

>  * Custom commands defined by user in packit config now are now run with shell, which makes it more convenient (by [@TomasTomecek](https://github.com/TomasTomecek), [#871](https://github.com/packit-service/packit/pull/871)).

Reviewer: ```suggestion
 * Custom commands defined by user in packit config now are run in the shell, no more `bash -c` required (by [@TomasTomecek](https://github.com/TomasTomecek), [#871](https://github.com/packit-service/packit/pull/871)).
```
